.devcontainer/Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM python:latest
+FROM python:3.10-alpine
+
+RUN addgroup --system dev && \
+    adduser --system --shell /usr/sbin/nologin --ingroup dev dev
 
-RUN useradd -ms /bin/bash dev
```

Comment:
Using alpine image there is another problem, that's the opposite, i even tried here, useradd and groupadd doesn't work, only adduser, and this may not be a problem since everyone will have the same environment.
And the group I add because it already creates one by default with adduser, so i just create it first by making it a system group

Reply:
My apologies. I didn't know this. Thanks! :+1:

Reply:
If we change I mean, read the above thread